doc/reference/reference_lua/popen.rst

```diff
@@ -100,7 +100,7 @@ Below is a list of all ``popen`` functions and handle methods.
              (if failure) ``nil, err``
 
     Possible errors: if a parameter is incorrect, the result is
-    "IllegalParams: incorrect type or value of a parameter".
+    `IllegalParams: incorrect type or value of a parameter`.
```

Comment:
please remove all kinds of quotes and back-ticks: this should be plain text; otherwise the descriptions still look like exact error messages

Reply:
fixed in 89e9596c310e981d07f3835e085ebb1e6dae9ac4